### ISSUE TITLE
Render man_made=surveillance for OsmAnd map style

### ISSUE THREAD

**Issue by letypequividelespoubelles** [osmandapp/OsmAnd#14974]
# 🚀 feature request

### Description

Display man_made=surveillance in OsmAnd map style. I'm pretty sure it used to be the case before v4.2. They are still displayed in topo map style in v.4.2.6.

### Describe the solution you'd like
Display man_made=surveillance. An option to display the view range of the camera (using camera:direction=0-360) tag would be wonderfull.

### Describe alternatives you've considered

None. Thanks for this amazing app.

**Comment by vshcherb** [comment]
I don't think it was ever on the map we probably only discussed to have it in POI
